src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -275,105 +285,63 @@ For an example of how to build an internal integration, see [our Round Robin Iss
 
 You can have up to 20 tokens at a time for any given internal integration.
 
-#### How to use for requests
+### Using Tokens for Requests
 
 When making requests to the Sentry API, you use the token just like you would when you're typically making [API requests](/api/auth/). Tokens are associated with the Sentry organization that created the integration (and therefore was automatically installed).
 
-#### Expiration
+### Token Expiration
 
 Tokens never expire, but you can manually revoke them.
 
 ![Image showing the ability to copy tokens, create new tokens, revoke tokens, and when they were created in UTC.](tokens.png)
 
-### Webhooks and Alerts
-
-Alerts are the same as public integrations -- see [Alerts](#alerts) for general information and see [Webhook Alerts](/product/integrations/integration-platform/webhooks/#issue-alerts) for more detail on the request and payload.
-
-### Integration Webhooks
-
-Since internal integrations are automatically installed (and uninstallation is essentially deleting the whole integration), there are no [un]installation webhooks. For more information, see the [full documentation on Webhooks](/product/integrations/integration-platform/webhooks/).
-
-## UI Components
-
-The Sentry Integration Platform provides the ability to add rich UI components to [sentry.io](https://sentry.io) itself through a  simple, declarative syntax.
-
-For more information, see the [full documentation on UI Components](/product/integrations/integration-platform/ui-components/).
-
-## Webhooks
-
-Webhooks allows your service to receive requests about specific resources, depending on your selection. For more information, see the [full documentation on Webhooks](/product/integrations/integration-platform/webhooks/).
-
-## Authorized Origins
-
-It is possible to use Auth Tokens from the browser if you allow the origins of the pages making the requests. In the field that is called `Authorized JavaScript Origins`, add each origin you want to be separated by a newline (for example, docs.sentry.io). You do not need the protocol in the origin (http or https). At this moment, you cannot use any wildcard characters (for example, `*.sentry.io`), so if you have multiple subdomains, you will need to add them individually.
 
 ## FAQ
 
-#### How are Sentry Integrations different from Plugins?
-
-Plugins (aka Legacy Integrations) are extensions of Sentry packaged as Python libraries. Plugins have been contributed both internally (for example, sentry-plugins) and externally by outside developers (for example, sentry-trello). Plugins are configured separately for each project.
-
-<Note>
-
-Going forward, writing a Sentry Integration is the preferred method of integrating with Sentry."
-
-</Note>
-
-#### How is the Sentry Integration different from OAuth Apps?
+#### How is a Sentry Integration different from OAuth Apps?
```

Comment:
All these FAQs are at an H4, but we can bring them up to H3 and we shouldn't be skipping heading levels. Is there any reason you did that here or just an oversight?

```suggestion
### How is a Sentry integration different from OAuth apps?
```

Reply:
I left them as H4s because the titles are so long the sidebar felt cluttered, I can make them H3s though.